package.json

```diff
@@ -34,7 +34,11 @@
     "cross-env": "^7.0.3",
     "cssnano": "^4.1.11",
     "del": "^6.1.1",
-    "eslint": "^6.8.0",
+    "eslint": "^8.23.1",
+    "eslint-config-standard": "^17.0.0",
```

Comment:
Could we reflect that we're also adding these new packages and give a brief explanation why in the commit message?

Reply:
✅ PR description updated https://github.com/alphagov/govuk-frontend/pull/2849#issue-1374123200

Reply:
Thanks for that!

I think what I'm trying to guard against is the case we've hit a coupla times, where we've had an unneeded dependency lying around because we made a change and didn't realise it was no longer needed. These plugins feel a bit "hidden" because they're not being directly imported anywhere in our code. I'm not sure if this is a big deal, or me just being overly nervous.